lib/utils/async-ember-test-helpers.js

```diff
@@ -0,0 +1,15 @@
+const ASYNC_EMBER_TEST_HELPERS = [
+  'blur',
+  'click',
+  'doubleClick',
+  'fillIn',
+  'focus',
+  'tap',
+  'triggerEvent',
+  'triggerKeyEvent',
+  'typeIn',
+  'visit',
+  'wait',
```

Comment:
I'm not sure where `wait` comes from. It's not mentioned in https://github.com/emberjs/ember-test-helpers/blob/master/API.md

Reply:
@Turbo87 ya, its `import wait from 'ember-test-helpers/wait'` (tldr; its "legacy" from pre `@ember/test-helpers` days)